src/gcp/cloudfunctionsv2.ts

```diff
@@ -467,10 +469,26 @@ export function functionFromEndpoint(
   );
   // Memory must be set because the default value of GCF gen 2 is Megabytes and
   // we use mebibytes
-  const mem: number = endpoint.availableMemoryMb || backend.DEFAULT_MEMORY;
+  const mem = endpoint.availableMemoryMb || backend.DEFAULT_MEMORY;
   gcfFunction.serviceConfig.availableMemory = mem > 1024 ? `${mem / 1024}Gi` : `${mem}Mi`;
   proto.renameIfPresent(gcfFunction.serviceConfig, endpoint, "minInstanceCount", "minInstances");
   proto.renameIfPresent(gcfFunction.serviceConfig, endpoint, "maxInstanceCount", "maxInstances");
+  proto.renameIfPresent(
```

Comment:
The old code defaulted to 80 concurrents if concurrency was not set and CPU was >= 1. We should do the same still.

Reply:
The `resolveCpuAndConcurrency` function in the prepare phase already provides the default — are there any edge cases in which the prepare phase does not happen, but the release phase does?

Reply:
It's fine. I may have just been programming more defensively than necessary in previous versions (or, more likely, it became clear that resolveCpuAndConcurrency had to be written to support inferencing on update after I had already written the fabricator code)